# Internal patch.
# Don't print the library path since it's default anyway.
--- OpenEXR.pc.in	2007-03-30 10:14:05.000000000 -0700
+++ OpenEXR.pc.in	2015-02-01 21:21:29.699453372 -0800
@@ -7,5 +7,5 @@
 Name: OpenEXR
 Description: OpenEXR image library
 Version: @OPENEXR_VERSION@
-Libs: -L${libdir} @ILMBASE_LDFLAGS@ -lIlmImf -lz @ILMBASE_LIBS@
-Cflags: @ILMBASE_CXXFLAGS@ -I${OpenEXR_includedir}
+Libs: -lIlmImf -lz @ILMBASE_LIBS@
+Cflags: -I${OpenEXR_includedir}
